src/main/java/com/chessmates/service/EntityServiceImpl.groovy

```diff
@@ -2,130 +2,35 @@ package com.chessmates.service
 
 import com.chessmates.model.Game
 import com.chessmates.model.Player
-import com.chessmates.utility.GetPageFunction
-import com.chessmates.utility.LichessApi
-import com.chessmates.utility.LichessResultSet
-import org.apache.commons.lang3.tuple.ImmutablePair
 
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
 
-import java.util.stream.Collectors
-
 /**
  * Service handling data requests relating to Lichess entities.
  */
 @Service
 class EntityServiceImpl implements EntityService {
 
-    static final String TEAM_NAME = 'scott-logic'
-
-    private LichessApi lichessApi
-    // TODO: Temporary - this wants to come from the database.
-    private Map<ImmutablePair, Game> playerIdToLatestGame = new HashMap<>() // A mapping of player IDs to their latest process game.
+    private LichessDataService lichessDataService
 
     @Autowired
-    EntityServiceImpl(LichessApi lichessApi) {
-        this.lichessApi = lichessApi
+    EntityServiceImpl(LichessDataService lichessDataService) {
+        this.lichessDataService = lichessDataService
     }
 
     /**
-     * All SL players.
+     * Get all Lichess players.
      */
     @Override
-    List<Player> getPlayers() {
-        def fetchPlayerPage = { String teamId, int pageNum -> lichessApi.getPlayers(teamId, pageNum) }
-
-        def neverStop = { t -> false}
-
-        def resultSet = new LichessResultSet<Player>(
-                (GetPageFunction)fetchPlayerPage.curry(TEAM_NAME),
-                neverStop
-        )
-
-        resultSet.get()
-    }
+    // TODO: Replace with fetch from DB.
+    List<Player> getPlayers() { lichessDataService.getPlayers(null) }
```

Comment:
How about we make a version of this method that takes no parameter rather than passing null